Defs/Ammo/Rifle/6.5x52mm Carcano.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+	<!-- ========== Based off of 7.92x57mm Mauser ========== -->

```

Comment:
I don't know that it matters much, but I'd use the existing 6.5mm Arisaka for stat reference rather than the Mauser. They're dimensionally very similar rounds.